performance.php

```diff
@@ -4,3 +4,4 @@
 require_once( __DIR__ . '/performance/bulk-edit.php' );
 require_once( __DIR__ . '/performance/vip-tweaks.php' );
 require_once( __DIR__ . '/performance/do-pings.php' );
+require_once( __DIR__ . '/performance/plugins.php' );
```

Comment:
I would suggest avoiding `glob` in favor of hardcoded file inclusion. `glob` is convenient, but makes the app's behavior more 'magical' and less predictable.

It's also something we look for in client code: https://vip.wordpress.com/documentation/vip/code-review-what-we-look-for/#implicit-file-includes